eng/pipelines/coreclr/templates/helix-queues-setup.yml

```diff
@@ -76,23 +76,23 @@ jobs:
     # Linux musl x64
     - ${{ if eq(parameters.platform, 'linux_musl_x64') }}:
       - ${{ if eq(variables['System.TeamProject'], 'public') }}:
-        - (Alpine.314.Amd64.Open)Ubuntu.1804.Amd64.Open@mcr.microsoft.com/dotnet-buildtools/prereqs:alpine-3.14-helix-amd64
+        - (Alpine.315.Amd64.Open)Ubuntu.1804.Amd64.Open@mcr.microsoft.com/dotnet-buildtools/prereqs:alpine-3.15-helix-amd64
```

Comment:
We had the "newer" Alpine here. Should we then rather use 3.17 here?
Based on https://endoflife.date/alpine 3.15 will get EOLed around the time we ship 8.0.

Reply:
I'm not sure and I did not want to jump too far. Up to 3.16 Alpine uses OpenSSL 1.1 while 3.17 switched to 3.0. It may be beneficial to run both - at least for now. ASP.NET jumped on 3.16 and we may do that later before 8.0 ships. For now, I'm proposing to run on oldest(ish) supported version.